docs-conceptual/azps-7.1.0/azps-versioning-release-cadence.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Azure Az PowerShell module versioning and release cadence
+description: This article contains versioning and release cadence information for the Azure Az PowerShell module.
+ms.devlang: powershell
+ms.topic: conceptual
+ms.date: 01/31/2022
+ms.custom: devx-track-azurepowershell
+ms.service: azure-powershell
+---
+
+# Azure Az PowerShell module versioning and release cadence
+
+The Azure Az PowerShell module is a rollup module containing more than seventy generally available
+(GA) service modules for managing Azure resources directly from PowerShell.
+
+## Versioning
+
+The Az PowerShell modules follow [Semantic Versioning](https://semver.org/) for version numbering.
+Versions of Azure service modules fall into one of three categories:
+
+- Generally available. Modules version 1.0.0 and higher without _preview_ in the version. Adheres to
+  breaking change policy.
+- Preview. Modules less than version 1.0.0. Do not adhere to breaking change policy.
+- Feature preview. Modules version 1.0.0 and higher with _preview_ in the version. Do not adhere to
+  breaking change policy.
+
+There are two Az PowerShell rollup modules:
+
+- Az. Installs all GA service modules for managing Azure resources.
+- AzPreview. Installs all GA and preview modules for managing Azure resources. Does not include
+  feature preview modules.
```

Comment:
Should we have a link to the modules in the gallery for reference? 
We should also mention the consistency of version number between Az and AzPreview.